### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by superzzp**

### CODE REVIEW

File: src/components/Authentication/Login.js [hunk 1]
```diff
@@ -1,53 +1,145 @@
...
+              </div>
+              Sign In with Google
             </Button>
-        </div >
-    )
-
+          </CardContent>
+          <Divider variant="middle" />
+          <CardActions className={classes.action}>
+            <Typography className={classes.textNotice} align="center">
```
Comment (ianmah) [inline src/components/Authentication/Login.js:129]: We can remove this

File: src/components/Authentication/Login.js [hunk 2]
```diff
@@ -1,53 +1,138 @@
...
+    padding: "20px"
+  },
+  button: {
+    margin: theme.spacing(1)
+  },
+  action: {
+    display: "flex",
+    justifyContent: "space-around"
+  },
+  textNotice: {
```
Comment (ianmah) [inline src/components/Authentication/Login.js:69]: Don't need this right

Comment (superzzp) [reply src/components/Authentication/Login.js]: That's a good catch, removed in latest commit

File: src/components/Authentication/Login.js [hunk 3]
```diff
@@ -1,57 +1,133 @@
...
+    display: "flex",
+    justifyContent: "space-around"
+  }
+}));
 
-    return (
-        <div>
-            <Helmet>
-                <title>UBC BizTech - Log In or Sign Up</title>
-            </Helmet>
```
Comment (ianmah) [inline src/components/Authentication/Login.js:38]: We need to keep this part

Comment (superzzp) [reply src/components/Authentication/Login.js:38]: fixed in the latest commit

Comment (superzzp) [reply src/components/Authentication/Login.js:38]: thanks ian

File: src/components/Authentication/Login.js [hunk 4]
```diff
@@ -1,57 +1,137 @@
...
+        alert("You must use a ubcbiztech.com email");
+      }
+    })
+    .catch(err => console.log("Not signed in:", err));
+  return (
+    <Container component="main" maxWidth="xs">
+      <CssBaseline />
+      <div className={classes.paper}>
+        <Card className={classes.root} width="400px">
+          <Helmet>
```
Comment (ianmah) [inline src/components/Authentication/Login.js:92]: I think it's best for visibility if we put the helmet as close to the top of the component as possible, so maybe next to/above the `CssBaseline`?

**Pull request merged**